.github/ISSUE_TEMPLATE/story.md

```diff
@@ -6,35 +6,58 @@ labels: 'story'
 assignees: ''
 
 ---
-
-## User story 
-
-<!-- Always has an estimation. Always drives business value.  Always gets QA'd.  Always fits within 1 sprint. -->
+## Problem
 
 TODO
-
 <!-- 
-Describe in a way of a user story what needs to be done, who wants it and for what purpose.
-Use this format:
-"As a _________, I want ________________."
-e.g. "As an admin I would like to be asked for confirmation before deleting a user so that I do not accidentally delete a user."
+Motivate this feature by describing what is the user's problem. 
+- Who are the affected users? 
+- What is the current situation? Why does the current situation hurt? 
+- What are they doing right now to resolve this issue? Why is this so bad?
+- How many users does this affect? How do you know? Share metrics or links to videos. 
+-->
 
+TODO
+As a _________, 
+I want to ________________, 
+so that I can___________."
+
+<!--
 Things to consider:
-- Who is the human? (`As an observer…`)
 - What screen are they looking at?  (`As an observer on the host details page…`)
 - What do they want to do? (`As an observer on the host details page, I want to run a permitted query.`) 
-- What is the current situation? Why does the current situation hurt? 
+-->
+
+## Business Case
+
+TODO
+<!--
+Why should Fleet work on this problem? How does this contribute to our reaching our strategic goals?
+-->
+
+## Measurement
+TODO
+<!--
+- How will we know that the problem has been solved or improved? 
+- What is the current state of the measurement?
+- What measurements do you need to implement? 
 -->
 
 ## Requirements
 
+TODO
+1.
+2.
+3.
+
 <!-- Things we tend to forget about -->
-- **Documentation** If the API is changing, then the [REST API docs](https://fleetdm.com/docs/using-fleet/rest-api) will need to be updated.
```

Comment:
"Reference documentation".  It can be part of the main documentation bullet.  But explicitly calling out and linking to the REST api docs is the goal of that.

Or this part could be consolidated with the REST API changes section below.  Let's keep the link.